src/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapPlot.jsx

```diff
@@ -154,11 +98,46 @@ const HeatmapPlot = (props) => {
     mouseOver: handleMouseOver,
   };
 
+  if (!selectedGenes || selectedGenes.length === 0) {
+    return (
+      <center>
+        <Empty
+          description={(
+            <span>
+              Please add gene(s) to the heatmap from the Gene list tool
+            </span>
+          )}
+        />
```

Comment:
Using "please" in instructions like this is not recommended in UX, it makes the interaction a bit stuffy.

Typically a good empty state has the following:

* a header describing *what* it is you're meant to see here [e.g. gene expression data], and making it clear that there is an action you can perform to do it
* give you hints on how to do it in a friendly, conversational tone

See also

> Good, modern empty state designs I’ve observed are typically two parts instruction, one part delight. If a UI is empty, it’s the UX designer’s job to help users understand what will go in the space when it’s populated. If it’s the user’s job to populate it, the messaging should include instructions about how. (For example, an empty “Classes” table in an edtech product might include copy informing teachers how to create their first class.)

Potentially better: 

```
                <Empty description={(
                  <>
                    <div><Text type='primary'>No expression data to show</Text></div>
                    <div><Text type='secondary'>You can add genes to display here from the gene list tool.</Text></div>
                  </>
                )}
                />
```